generators/app/index.js

```diff
@@ -71,31 +75,67 @@ module.exports = class extends Generator {
     }
   }
 
+  _showBusy(statusText) {
+    if (this.busyIndicatorTimer) {
+      this.hideBusy();
+    }
+    const output = ['.  ', '.. ', '...'];
+    let i = 0;
+    process.stdout.write(`\r${statusText}   `);
+    this.busyIndicatorTimer = setInterval(() => {
+      process.stdout.write(`\r${statusText}${output[i++]}`);
+      i %= output.length;
+    }, 250);
+  }
+
+  _hideBusy() {
+    if (this.busyIndicatorTimer) {
+      clearInterval(this.busyIndicatorTimer);
+      delete this.busyIndicatorTimer;
+      process.stdout.write(`\n`);
+    }
+  }
+
   async prompting() {
     this.log(yosay(`Welcome to the ${chalk.red("easy-ui5")} generator!`));
 
+    // create the octokit client to retrieve the generators from GH org
     const octokit = new Octokit({
       userAgent: `${this.rootGeneratorName()}:${this.rootGeneratorVersion()}`,
       auth: this.options.ghAuthToken,
     });
 
-    const reqRepos = await octokit.repos.listForOrg({
```

Comment:
Had some issues when quota exceeded for amount of request while running local tests without authentication - so I added proper error handling